pkg/restapi/edv/models/models.go

```diff
@@ -60,3 +60,10 @@ type Query struct {
 	Name  string `json:"index"`
 	Value string `json:"equals"`
 }
+
+// LogSpec represents a request to change the logging level.
+// Format: ModuleName1=Level1:ModuleName2=Level2:ModuleNameN=LevelN:AllOtherModuleDefaultLevel
+// Example: restapi=debug:edvrest=info:info
+type LogSpec struct {
+	Spec string `json:"spec"`
```

Comment:
this would be more API-friendly if `Spec` were an array of some object tuple with "module" and "level" attributes

Reply:
I agree - however, I chose this syntax in order to match what Fabric uses. This way the Ops team can use the same syntax for both.